var/spack/repos/builtin/packages/extrae/package.py

```diff
@@ -79,7 +81,10 @@ class Extrae(AutotoolsPackage):
     build_directory = "spack-build"
 
     variant("dyninst", default=False, description="Use dyninst for dynamic code installation")
-    depends_on("dyninst@:9", when="+dyninst")
+    with when("+dyninst"):
+        depends_on("dyninst@10.1.0:")
```

Comment:
Does the new  version range apply to *all* of the listed versions or from some version on?

Reply:
I tested all versions. Should have mentioned that.